src/lib/preferences/preferences.cpp

```diff
@@ -214,7 +214,13 @@ Preferences::Preferences(BrowserWindow* window)
     //APPEREANCE
     settings.beginGroup("Browser-View-Settings");
     ui->showStatusbar->setChecked(settings.value("showStatusBar", false).toBool());
+    // NOTE: instantBookmarksToolbar and showBookmarksToolbar cannot be both enabled at the same time
```

Comment:
@nowrep 
Actual OS X testing and logic flow shows that this should be okay... unless you want to force a focus event on browser startup as a "reminder".